common/src/main/java/org/keycloak/common/Profile.java

```diff
@@ -395,6 +397,13 @@ public static boolean isFeatureEnabled(Feature feature) {
         return getInstance().features.get(feature);
     }
 
+    public static void enableFeature(Feature feature) {
+        var featureEnabled = getInstance().features.get(feature);
+        if (!featureEnabled) {
+            getInstance().features.put(feature, true);
+        }
```

Comment:
Not super happy to do it, but I didn't find any better approach.

Reply:
This approach is not thread safe. You can choose to use a concurrent hash map, or a volatile field where you replace the old map with a new map.